src/Xamarin.Android.Cecil/Xamarin.Android.Cecil.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CecilDirectory>$(MSBuildThisFileDirectory)\..\..\external\cecil</CecilDirectory>
+    <CecilPreparedFlag>prepared.flag</CecilPreparedFlag>
+    <OutputPath Condition="'$(OutputPath)' == ''">bin\$(Configuration)</OutputPath>
+    <CecilOutputPath>$([System.IO.Path]::GetFullPath ('$(OutputPath)'))</CecilOutputPath>
+  </PropertyGroup>
+  <Target Name="PrepareCecil"
+      Inputs="$(CecilDirectory)\Mono.Cecil.sln"
+      Outputs="$(CecilPreparedFlag)">
+    <Exec Command="cd $(CecilDirectory); ln -sf $(MSBuildThisFileDirectory)\AssemblyInfo.cs Mono.Cecil.AssemblyInfo.cs" />
```

Comment:
OK, I used Copy instead and added the files to Inputs so that the target is rebuilt when these are changed.